chaste_codegen/_lookup_tables.py

```diff
@@ -73,12 +74,15 @@ def __init__(self, model, lookup_params=DEFAULT_LOOKUP_PARAMETERS):
         self._method_printed = None
 
         for param in self._lookup_parameters:
+            print(param['metadata_tag'])
             try:
                 var = self._model.get_variable_by_ontology_term((OXMETA, param['metadata_tag']))
                 self._lookup_variables.add(var)
                 param['var'] = var
             except KeyError:
-                pass  # variable not tagged in model
+                # variable not tagged in model
+                LOGGER.warning('A lookup table was specified for ' + param['metadata_tag'] +
+                               ' but it is not tagged the model, skipping!')
```

Comment:
```suggestion
                LOGGER.warning('A lookup table was specified for ' + param['metadata_tag'] +
                               ' but it is not tagged in the model, skipping!')
```
Don't think we need the comment to replicate the warning!